src/components/DisqualificationTimerBar/DisqualificationTimerBar.tsx

```diff
@@ -33,27 +37,71 @@ const DisqualificationTimerBar: React.FC<DisqualificationTimerBarProps> = ({
   activeStep,
   modalOpen,
   failingSteps,
+  beepConfig,
 }) => {
   const getMaxTime = useMemo(() => {
     if (!failingSteps.length) return STEP_VS_MESSAGE_MAPPING[activeStep].time;
     
     return Math.max(
-      ...failingSteps.map(step => STEP_VS_MESSAGE_MAPPING[step].time)
+      ...failingSteps.map(step => STEP_VS_MESSAGE_MAPPING[step].time) 
     );
   }, [failingSteps, activeStep]);
 
   const message = STEP_VS_MESSAGE_MAPPING[activeStep].message;
   const proctor = useAppSelector(selectProctor);
 
   const [timeLeft, setTimeLeft] = useState(getMaxTime);
+  const audioRef = useRef<HTMLAudioElement | null>(null);
 
   useEffect(() => {
     setTimeLeft(getMaxTime);
-  }, [modalOpen, getMaxTime]);
+
+    console.log("Reached here");
+
+    if (beepConfig?.enabled && beepConfig.sounds[activeStep]) {
+      
+      if (!beepConfig.sounds[activeStep]) {
+        console.error('Audio file path is empty or invalid');
```

Comment:
Remove error/warning logs. The lib should work without these files as well. It's just an add-on.